protos/io_descriptors.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+/* Value represents a single instance of the supported datatypes. */
+message Value{
+	oneof types{
+		bytes b=1;
```

Comment:
understood. lets make the changes from "b" to "bytes_" @sauyon ?

Reply:
Sounds good to me.

Reply:
In numpy, timedelta is represented as `m` and datetime is `M`. If the naming convention stays consistent, the protobuf has to also be `m` and `M` respectively. This is not possible however because proto3 doesn't support this.
Error❌: `The JSON camel-case name of field "m" conflicts with field "M". This is not allowed in proto3.`
How should I name these fields?

Reply:
Maybe we should give up on using numpy names and just use protobuf standard ones then. @ssheng thoughts?

Reply:
For example: `int32: int32_`,`string: string_`, `bool: bool_` and so on...

Reply:
how about `m_` and `M_`?

Reply:
`m_` and `M_` don't work either. When converted to lowerCamelCase, if the field names are equal then it doesn't work.

Reply:
Let's just go with the protobuf names for now.